### PR TITLE
AP_InertialSensor: added support for IIM-42652 and ICM-40605 sensors

### PR DESCRIPTION
**Pull request by QioTek**
AP_InertialSensor: added support for IIM-42652 and ICM-40605 sensors

### CODE REVIEW

File: libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp [hunk 1]
```diff
@@ -3,12 +3,12 @@
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
-
+
```
Comment (hendjoshsr71) [inline libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp:6]: nitpick: extra white space

File: libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.
-
+   
    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.
-
+
```
Comment (hendjoshsr71) [inline libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.cpp:11]: nitpick: extra white space

File: libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.h [hunk 3]
```diff
@@ -49,8 +51,8 @@ class AP_InertialSensor_Invensensev3 : public AP_InertialSensor_Backend
 
     void set_filter_and_scaling(void);
     void fifo_reset();
-
-    /* Read samples from FIFO */
+
```
Comment (hendjoshsr71) [inline libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.h:54]: nitpick: extra white space

File: libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.h [hunk 4]
```diff
@@ -49,8 +51,8 @@ class AP_InertialSensor_Invensensev3 : public AP_InertialSensor_Backend
 
     void set_filter_and_scaling(void);
     void fifo_reset();
-
-    /* Read samples from FIFO */
```
Comment (hendjoshsr71) [inline libraries/AP_InertialSensor/AP_InertialSensor_Invensensev3.h:53]: nitpick: you removed one space here

**Review by hendjoshsr71 — CHANGES_REQUESTED** [review]
LGTM: besides the small whitespace errors

I will take it to the call to make sure everyone else is ok with the structure.

**Comment by hendjoshsr71** [comment]
Oh one important GIT note. You should never commit changes to your `master branch` like you did here. You should create a branch `add_iim42652_imu` for example and commit to that branch.

**Comment by tridge** [comment]
I have fixed up the whitespace issues

**Review by tridge — APPROVED** [review]

**Comment by QioTek** [comment]
> Oh one important GIT note. You should never commit changes to your `master branch` like you did here. You should create a branch `add_iim42652_imu` for example and commit to that branch.

Thanks for reminding me. I'll create a new branch next time.

**Pull request merged**